marimo/_runtime/packages/module_name_to_pypi_name.py

```diff
@@ -663,6 +663,7 @@ def module_name_to_pypi_name() -> dict[str, str]:
         "rsbackends": "RSFile",
         "ruamel": "ruamel.base",
         "saga": "saga-python",
+        "sage": "passagemath-standard",
```

Comment:
Thanks for the PR. Isn't the standard distribution `sagemath-standard`?

Reply:
Ah, ok. We can also pre-package the bins for molab, but I think the official sage distribution is what most users would expect

Reply:
The official Sage distribution is not compatible with installation via pip / uv.